examples/siro_sandbox/controllers/baselines_controller.py

```diff
@@ -291,19 +293,68 @@ def get_cartesian_obj_coords(self, obj_trans, env):
 
         return rho, phi
 
+    def get_geodesic_distance_obj_coords(self, obj_trans, env):
+        # TODO: we use geodesic_distance here
+        articulated_agent_T = self.get_articulated_agent().base_transformation
+        rel_pos = articulated_agent_T.inverted().transform_point(obj_trans)
+        rho, phi = cartesian_to_polar(rel_pos[0], rel_pos[1])
```

Comment:
This code duplicates `get_cartesian_obj_coords` function. Can we call `get_cartesian_obj_coords` to get `rho` and `phi` instead?